webpack/components/TemplateSyncResult/components/SyncResultList.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { ListView } from 'patternfly-react';
 import PropTypes from 'prop-types';
 
-import Pagination from 'foremanReact/components/Pagination/PaginationWrapper';
+import Pagination from 'foremanReact/components/Pagination';
```

Comment:
can you also update the components new props?
```jsx
<Pagination
      itemCount={templates.length}
      {...pagination} // not sure if needed
      onChange={pageChange}
    />
```

Reply:
updated